src/ApiStatusApi.ts

```diff
@@ -0,0 +1,49 @@
+import fetch from "node-fetch";
+
+type RegionState = "up" | "troubled" | "down";
+
+export interface RegionStatus {
+    "uptime": string,
+    "performance": string,
+    "state": RegionState
```

Comment:
Normally we only quote keys if they contain characters that aren't normally allowed (like `-`).